src/main/java/com/dev/theater/Main.java

```diff
@@ -37,5 +45,10 @@ public static void main(String[] args) {
         movieSessionService.add(movieSession);
         movieSessionService.findAvailableSessions(movie.getId(),
                 LocalDate.now()).forEach(System.out::println);
+        User user = authenticationService.register("test@bigmir.net", "12345");
+        System.out.println(user.toString());
+        System.out.println(userService.findByEmail("test@bigmir.net").toString());
+        authenticationService.register("test2@bigmir.net", "12345");
```

Comment:
why are you registering the same user twice?

Reply:
they are different. test and test2